tests/e2e/backup_restore_suite_test.go

```diff
@@ -170,7 +186,7 @@ var _ = Describe("AWS backup restore tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			// wait for backup to not be running
-			Eventually(IsBackupDone(dpaCR.Client, namespace, backupName), timeoutMultiplier*time.Minute*4, time.Second*10).Should(BeTrue())
+			Eventually(IsBackupDone(dpaCR.Client, namespace, backupName), timeoutMultiplier*time.Minute*12, time.Second*10).Should(BeTrue())
```

Comment:
Was this test failing regularly because of the timeout? A jump from 4 minutes to 12 minutes seems large

Reply:
Regularly times out for 4.11 aws csi at least.

Reply:
Picked 12 from velero backup default timeout of 10 and add 2 minutes for maybe api server result delays.